tests/appservice/test_scheduler.py

```diff
@@ -40,9 +41,6 @@
 
 from ..utils import MockClock
 
-if TYPE_CHECKING:
-    from twisted.internet.testing import MemoryReactor
```

Comment:
This isn't *really* related to this, but we shouldn't be importing from `twisted.internet.testing`.

Reply:
Do you mean because `twisted.internet.testing` doesn't exist on our oldest supported Twisted version, or is there some other reason here?

Reply:
Yes, that's all. 👍